python/oneflow/test/modules/test_median.py

```diff
@@ -56,6 +56,12 @@ def test_median_reduce_one_dim_0size(test_case):
         x = random_tensor(ndim=3, dim1=0, requires_grad=False).to(device)
         return torch.median(x, 0)
 
+    @autotest(n=5, auto_backward=False, check_graph=False)
```

Comment:
```suggestion
    @autotest(n=5)
```
这里应该测试 backward 和 graph ，为什么把它们关掉了呢？